.github/workflows/reproduce.yml

```diff
@@ -43,6 +43,13 @@ jobs:
           git clone --no-checkout git://develop.git.wordpress.org/ .
           git checkout "tags/${TAG}" -b "${TAG}"
 
+      - name: Install SVN
+        if: inputs.source == 'develop.svn.wordpress.org'
+        run: | #shell
+          sudo apt-get update
```

Comment:
No need for `update`. It is automatic.

Reply:
Unfortunately [that's not true on GHA runners](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/customizing-github-hosted-runners#installing-software-on-ubuntu-runners).

Reply:
All right.
I think updating takes time and installation won't fail without it.